handbook/business-operations/README.md

```diff
@@ -518,6 +522,7 @@ Once the new team member replies and accepts their offer in writing, Head of Bus
        - **Start date** _(the new fleetie's first day, YYYY-MM-DD)_
        - **Location** _(derive this from the physical address)_
        - **GitHub username**  _(username of 2FA-enabled GitHub account)_
+       - **@fleetdm.com email** _(update with email this person should have)_
```

Comment:
set this to your best guess of what the person should have

Reply:
actually your way is better- I was just trying to use the word "set" so it matches the "offer sent" one below